external/windows64/README.md

```diff
@@ -63,40 +162,77 @@ Extract them all into `%NUPIC_CORE_SOURCE%/build` directory. You may need to un-
+
+#### MinGWPy GCC only
+
+##### Apache Portable Runtime (APR)
+
+Install the 64 bit version of MSys2 - http://sourceforge.net/p/msys2/wiki/MSYS2%20installation/
+
+From within the MSys2 environment, run the `update-core` script followed by `pacman -Su`. Using `pacman -Ss apr` searches for APR packages. Use the following commands to install the required APR libraries; `pacman -S mingw64/mingw-w64-x86_64-apr --noconfirm` and `pacman -S mingw64/mingw-w64-x86_64-apr-util --noconfirm` 
+
+This installs (assuming a default installation directory for MSys2) the libraries into `C:/msys64/mingw64/lib`. The two APR libraries (`libapr-1.a` and `libaprutil-1.a`) can then be copied into the `%NUPIC_CORE%\external\windows64-gcc-\lib` directory. And used with the CMake build command as described above to rebuild the core library.
+
+##### Cap'n Proto
+
+This is part of the main nupic.core CMake build process. The appropriate version is git cloned, as well as the associated Cap'n Proto compiler tools. It is built using the `MinGW Makefile` and ends up in the `%NUPIC_CORE%\build\scripts\ThirdParty\Install` directories.
+
+##### Yaml, Yaml-cpp, and Z libbraries
+
+A CMake command, or the CMake-GUI program, can be used to generate MinGW Makefiles. These can then be used with the `mingw32-make -f Makefile install` command to rebuild these libraries. Make sure that the `CMAKE_BUILD_TYPE` and `CMAKE_INSTALL_PREFIX` are set appropriately.
+
+#### Microsoft Visual Compilers only
 
-### Apache Portable Runtime (APR)
+##### Apache Portable Runtime (APR)
 
 apr.dsw and aprutil.dsw workspace files can be imported into Visual Studio 2015. Directory naming allows these solutions to find the apr-iconv projects. With APR we only require the 'apr' project to be built. The 'libapr' project is for a DLL version. And with Apr-util we just need to build the 'apr-util' project.
 
-### Cap'n Proto
+##### Cap'n Proto
 
-Download version **0.5.2** from https://capnproto.org/capnproto-c++-win32-0.5.2.zip 
+Download version **0.5.3** from https://capnproto.org/capnproto-c++-win32-0.5.3.zip and extract into %NUPIC_CORE%\build directory.
 
-The three executable files found in `%NUPIC_CORE_SOURCE%\build\capnproto-tools-win32-0.5.2` should match the corresponding `%NUPIC_CORE_SOURCE%\external\windows64\bin` executable files, and tie in with the external capnp and kj common include directories. 
+Install instructions can be found at https://capnproto.org/install.html Below is an example Visual Studio Command Prompt instructions to invoke cmake and generator a solution and project files for Cap'n Proto.
 
-Install instructions can be found at https://capnproto.org/install.html This is an example Visual Studio Command Prompt line to invoke cmake and generator a solution and project files for Cap'n Proto.
+<pre>
+cd %NUPIC_CORE%\build\capnproto-c++-win32-0.5.3\capnproto-c++-0.5.3  
+vcvarsall.bat  
+cmake -G "Visual Studio 14 2015 Win64"  
```

Comment:
One sections is for GCC only, the other for MSVC.
The GCC section (https://github.com/numenta/nupic.core/pull/754/files#diff-5cb6e82904d0242c10b78fac8c92f82fR216) tries to describe that capnp (main way to rebuild all of core and bindings) is now part of Core CMake (ExternalProject).
This MSVC section just replicates the capnp.org install instructions, but with respect to nupic core. The "Visual Studio 2015..." cmake generator spits out MSVC solution and project files that can be used to rebuild .lib files for MS Visual C compilers.